### PR TITLE
openapi generator updates

### PR DESCRIPTION
**Pull request by clowenhg**

### CODE REVIEW

File: generator/config.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
```
Comment (pmenglund) [inline generator/config.json:3]: we shouldn't use two spaces for indentation, as go uses tabs - I think I had `gofmt` in the script to make sure the code is correctly formatted before we commit it

**Review by pmenglund — CHANGES_REQUESTED** [review]

File: generator/config.json (hunk 1 above)
Comment (clowenhg) [reply generator/config.json:3]: That is specifically for THIS json file... for some dumb reason.
https://github.com/OpenAPITools/openapi-generator-cli#configuration

Comment (pmenglund) [reply generator/config.json:3]: WTAF 🤦

**Review by pmenglund — APPROVED** [review]
:shipit:

**Pull request merged**